Jetchat/app/src/main/java/com/example/compose/jetchat/data/SlidingWindow.kt

```diff
@@ -35,6 +35,8 @@ class SlidingWindow {
             // prepare output data structure
             var messagesInWindow = mutableListOf<ChatMessage>()
 
+            var messagesForSummarization = ""
```

Comment:
```suggestion
            val messagesForSummarization = ""
```

Reply:
I think this was a 'trick' because of the hardcoded test data. leaving it as `var`